src/settings-ui/Settings.UI/App.xaml

```diff
@@ -24,10 +24,10 @@
                 <Setter Property="Margin" Value="0,0,0,2" />
                 <Setter Property="Padding" Value="0,0,0,0" />
                 <Setter Property="HorizontalContentAlignment" Value="Stretch" />
-                <Setter Property="IsTabStop" Value="False"/>
```

Comment:
Will this have an impact on accessibility or keyboard navigation? Or was this property a no-op.

Reply:
this looks wrong, indeed

Reply:
thanks, probably accidental change